objects/BlessCurseManager.5933fb.ttslua

```diff
@@ -0,0 +1,475 @@
+-- Bless / Curse Manager
+-- updated by:      Chr1Z
```

Comment:
it's fine to leave in, but eventually this sort of information is all captured in github

Reply:
Yeah, will get rid of these remarks with the next release, but wanted to get all my local stuff online for now